### DISCUSSION TITLE
Mininum length right padding

### DISCUSSION THREAD

**Discussion by joshpetit** [polybar/polybar#2782] [Q&A]
So for the date module I am using this label:

```
label = %date:15%
```
Which resultes in a date that looks like this

![image](https://user-images.githubusercontent.com/59021155/183271317-16931a86-6786-4512-98da-956337310524.png)

Is there a way to get the padding for the minimum length to pad to the right instead of the left?

**Comment by patrick96** [comment]
Not for the token itself, but you can get padding to the right for the whole label: https://github.com/polybar/polybar/wiki/Formatting#label

**Reply by joshpetit** [reply]
Ah. I wanted to use the token because I have a toggle for date which shows the full date. 
![image](https://user-images.githubusercontent.com/59021155/183290657-01c51bf6-6c31-4c3d-b2d5-9ce280c27761.png)
So I have the min length of 15 in order to prevent the modules from resizing everytime I toggle it.

Would this be a valid feature request if it's not already possible? My initial thoughts is to make a min length of `-0` do right padding instead of left but that may be a bit not obvious 😅

**Reply by patrick96** [reply]
Yes, that's a valid feature request. The first thing I thought when I read the original post was "Hey! Don't we already have negative min-lengths for left-alignment?".

Turns out that we don't, but we do have zero-padding support by prefixing a `0`, which is probably what I was thinking about.

Anyway, feel free to open a feature request in the [ideas discussion tab](https://github.com/polybar/polybar/discussions/categories/ideas). I don't think having a negative min-length for that is unintuitive (but maybe that's just because I know how printf works)
